src/benchmarklib/listenable.cpp

```diff
@@ -0,0 +1,22 @@
+#include "listenable.hpp"
+
+namespace opossum {
+
+void Listenable::add_listener(const Event event, EventListener listener) {
```

Comment:
`listen(...)`?

Reply:
Isn't that the opposite? `Listenable::listen(...)` would mean that the `Listenable` is the one that listens for the event. But the object that is calling this method is actually the listener (or provides the listener), so it adds the listener to the Listenable with `add_listener`.

Reply:
At least it's ambiguous. Never mind.